pymedphys/tests/dicom/test_header_tweaks.py

```diff
@@ -45,7 +45,7 @@ def compare_dicom_cli(command, original, expected):
     pydicom.write_file(ORIGINAL_DICOM_FILENAME, original)
 
     try:
-        subprocess.check_call(command)
+        subprocess.check_call(" ".join(command), shell=True)
```

Comment:
not that its a problem, but what motivated this?

Reply:
Only an issue for Windows Python. Subprocess on windows when not using "shell=True" will sometimes have issues with being confined within a virtual environment. In this case it wasn't using the poetry version of Python to call pymedphys.